src/android/SecureStorage.java

```diff
@@ -295,10 +514,11 @@ private void handleLockScreenUsingNoOpOrSetNewPasswordIntent(IntentRequestType t
         Log.v(TAG, "Handling lock screen via no action or ACTION_SET_NEW_PASSWORD intent (Android 10 or newer)");
```

Comment:
My bad, didn't see this log here, but it also needs to be updated, similarly to the other.